runtime/ts/storage/firebase-storage.ts

```diff
@@ -13,6 +13,7 @@ import {KeyBase} from './key-base';
 import {atob} from '../../../platform/atob-web.js';
 import {btoa} from '../../../platform/btoa-web.js';
 import {CrdtCollectionModel} from './crdt-collection-model';
+import {Id} from '../id';
```

Comment:
in general we're going to need to refer to ts modules via the .js hack so we can preserve load-without-build for the shell. Not all imports require it (e.g. crdt-collection-model above seems to work OK) but I think we should probably start always doing it so that we can remain consistent?

This basically just means using `import {Id} from '../id.js';` here. Typescript understands js references to ts files but passes through the full reference (rather than the one without an extension), which then allows module loading to occur in the shell if the .js file is referenced without webPacking (or other ES6 shimming).

Reply:
Yeah, or I can do it in a cleanup patch later. Those went in before I realized what we needed to do for shell-facing classes.